tests/common/include/tests/common/tools/detector_metadata.hpp

```diff
@@ -98,54 +149,45 @@ struct full_metadata {
         e_rod = 1,
     };
 
-    // How to store and link materials
-    using material_definitions = tuple_vector_registry<material_ids, slab, rod>;
-    using material_link_type = dtyped_index<material_ids, dindex>;
+    /// How to store and link materials
+    template <template <typename...> class tuple_t = dtuple,
+              template <typename...> class vector_t = dvector>
+    using material_store = regular_multi_store<material_ids, empty_context,
+                                               tuple_t, vector_t, slab, rod>;
 
-    /// How many grids have to be built
-    enum grids : std::size_t {
-        n_other = kDefault,
-        n_z_phi_grids = kBrlGrids,
-        n_r_phi_grids = kEdcGrids
-    };
+    /// How to store and link materials
+    template <template <typename...> class tuple_t = dtuple,
+              typename container_t = host_container_types>
+    using binned_material_store = regular_multi_store<
```

Comment:
What is this for exactly?

Reply:
Nothing, yet. I will remove it, see comment above